src/core/thread/mle.hpp

```diff
@@ -780,6 +780,9 @@ class Mle : public InstanceLocator, private NonCopyable
     static constexpr uint8_t kMaxCriticalTxCount        = 6; // Max tx count for critical MLE message
     static constexpr uint8_t kMaxChildKeepAliveAttempts = 4; // Max keep alive attempts before reattach
 
+    static constexpr uint32_t kChildResetMinTimeoutMS    = 16000u;
+    static constexpr uint32_t kChildResetTimeoutJitterMS = 15000u;
```

Comment:
As in the comment in the code where these are used, we can adjust the values here to possibly tweak performance.